emu/decode.h

```diff
@@ -107,6 +107,30 @@ __no_instrument DECODER_RET glue(DECODER_NAME, OP_SIZE)(DECODER_ARGS) {
                            READMODRM; VXOR(xmm_modrm_val, xmm_modrm_reg,128);
                            break;
 
+                case 0x6e: TRACEI("movd modrm, xmm");
+                           // TODO: REX.W = 1 might be needed later
+                           READMODRM; VZLOAD(modrm_val, xmm_modrm_reg,32);
+                           break;
+
+                case 0x6f: TRACEI("movdqa xmm:modrm, xmm");
+                           READMODRM; VLOAD(xmm_modrm_val, xmm_modrm_reg,128); break;
+
+                case 0x73: READMODRM;
```

Comment:
Please put the READMODRM inside the switch, there might be opcodes that don't have a modrm

Reply:
nvm.

Reply:
no